safe-global/safe-wallet-web
# Refactor: fetch pending queue only in 1/N Safes

## What it solves

There's no need to fetch the untrusted queue in non-1/X Safes. In higher threshold Safes, all pending txs will be signed or delegated, so they'll be displayed in the main queue.

## How to test

1.
* Go to a 2+/X Safe
* Make and execute a tx
* Open the Network panel in the devtools
* Quickly go to the queue and observe the pending tx
* Observe that the title above the queue says "Next" (correct)
* Observe that NO `/queued?trusted=false` request is made

2.
* Go to a 1/X Safe
* Make and execute a tx
* Open the Network panel in the devtools
* Quickly go to the queue and observe the pending tx
* Observe that the title above the queue says "Pending"
* Observe that there IS a `/queued?trusted=false` request made
